app.py

```diff
@@ -0,0 +1,591 @@
+import os
+import math
+import json 
+import datetime
+
+# external
+from flask import Flask, request, jsonify
+from flask_socketio import SocketIO, send, emit, join_room, leave_room
+from flask_sqlalchemy import SQLAlchemy
+from flask_marshmallow import Marshmallow
+from sqlalchemy import (
+    Column,
+    Integer,
+    String,
+    Boolean,
+    ForeignKey,
+    DateTime,
+    Sequence,
+    Float,
+)
+import jwt
+
+import warnings
+# https://github.com/marshmallow-code/flask-marshmallow/issues/53
+with warnings.catch_warnings():
+     from flask_marshmallow import Marshmallow
+
+def debug(msg):
+    print('\n\n\n'+msg+'\n\n\n')
+
+
+app = Flask(__name__)
+basedir = os.path.abspath(os.path.dirname(__file__))
+
+app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///' + \
+    os.path.join(basedir, 'db.sqlite')
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+app.config['SECRET_KEY'] = '\xfa\xfe\xc5\x8d \xaf\xbe\xb8`\xbfz\xbc\x96Z\x9b\xc1\x9b]\xefB\x1a\x1b\xe7\xf4'
+
+
+db = SQLAlchemy(app)
+
+ma = Marshmallow(app)
+
+socketio = SocketIO(app, cors_allowed_origins='*')
+
+
+# Distance Calculator
+def distance(origin, destination):
+    """Find distance between two given points
+    params: (['latitudeA', 'longitudeA'], ['latitudeB', 'longitudeB'])
+    """
+
+    # Usage: distance([40.689202777778, -74.044219444444],
+    #             [38.889069444444, -77.034502777778])
+    #       or
+    # distance([latitudeA, longitudeA],
+    #             [latitudeB, longitudeB])
+    # return: distance in meters, round by 2
+
+    # Conversions:
+    # d_km = d_km/1000
+    # d_miles = d_km*0.621371
```

Comment:
add the usage and return description in the docstring itself. No need to add it in comments. You can add them like this:

  - **parameters**, **types**, **return** and **return types**::

          :param arg1: description
          :param arg2: description
          :type arg1: type description
          :type arg1: type description
          :return: return description
          :rtype: the return type description